azurerm/resource_arm_sql_administrator.go

```diff
@@ -0,0 +1,143 @@
+package azurerm
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/Azure/azure-sdk-for-go/services/sql/mgmt/2015-05-01-preview/sql"
+	"github.com/hashicorp/terraform/helper/schema"
+	uuid "github.com/satori/go.uuid"
+	"github.com/terraform-providers/terraform-provider-azurerm/azurerm/utils"
+	//"github.com/hashicorp/terraform/helper/validation"
+	//"github.com/terraform-providers/terraform-provider-azurerm/azurerm/helpers/response"
+)
+
+func resourceArmSqlAdministrator() *schema.Resource {
+	return &schema.Resource{
+		Create: resourceArmSqlAdministratorCreateUpdate,
+		Read:   resourceArmSqlAdministratorRead,
+		Update: resourceArmSqlAdministratorCreateUpdate,
+		Delete: resourceArmSqlAdministratorDelete,
```

Comment:
could we update the function name + these prefixes to `resourceArmSqlActiveDirectoryAdministrator`

Reply:
renamed as requested